src/scss/bitmovinplayer-ui.scss

```diff
@@ -1 +1,107 @@
-@import 'skin-modern/skin';
```

Comment:
Since we will only have one UI look and feel going forward (like now) there is no need anymore to add this extra level of 'abstraction'. I simply copied the content of `_skin.scss` file directly into this one. The code was not changed.